datasource/management/commands/groot-352.py

```diff
@@ -0,0 +1,109 @@
+import re
+from datetime import datetime
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.core.validators import URLValidator
+from django.utils import timezone
+
+from brand.models import *
+from datasource.models import *
+
+
+class Command(BaseCommand):
+    help = "This script enters information from usnic into brands for FCU entity type"
+
+    def handle(self, *args, **kwargs):
+        print("Running db_changes_fcu.py")
+        rssd_ids = []
+        with open(r"datasource\management\commands\cdfi_rssd_numbers.txt", "r") as file:
+            rssd_ids = [line.strip() for line in file]
+        filtered_usnic = Usnic.objects.filter(entity_type="FCU").values(
+            "ncua",
+            "website",
+            "country",
+            "legal_name",
+            "rssd",
+            "lei",
+            "fdic_cert",
+            "ein",
+            "occ",
+            "thrift_hc",
+            "cusip",
+        )
+        # selects the two fields to check for duplicates
+        brand_values = Brand.objects.values("tag", "name")
+        existing_tags = [x["tag"] for x in brand_values]
+        existing_names = [x["name"] for x in brand_values]
+        for row in filtered_usnic:
+            if row["legal_name"] in existing_names:
+                continue
+            else:
+                existing_names.append(row["legal_name"])
+            # getting the datetime django wants
+            naive_datetime = datetime.now()
+            aware_datetime = timezone.make_aware(naive_datetime)
+            new_name = row["legal_name"] + "_FCU"
+            try:
```

Comment:
This try except statement is unecessary. use `dict.get("key", "default")` instead.

Reply:
```
rssd = row.get("rssd")
if rssd in rssd_ids:
    new_name = row[3] + "_CDFI"
```